core/src/main/java/gyro/core/scope/RootScope.java

```diff
@@ -233,12 +239,12 @@ public Object convertValue(Type returnType, Object object) {
         return converter.convert(returnType, object);
     }
 
-    public List<Resource> findResources() {
-        return findResourcesIn(null);
+    public List<Resource> findSortedResources() {
+        return findSortedResourcesIn(null);
```

Comment:
If this method is not going to be overloaded, why not keep the original name `findResources`? `findSortedResources` seems like would have an overidden method `findResources`.